### PR TITLE
Fix the aot search regex

### PR DESCRIPTION
**Pull request by mkArtakMSFT**

### CODE REVIEW

File: .github/fabricbot.json [hunk 1]
```diff
@@ -3687,7 +3687,7 @@
             {
               "name": "bodyContains",
               "parameters": {
-                "bodyPattern": "(aot)/i",
+                "bodyPattern": "([Aa][Oo][Tt])+",
```
Comment (amcasey) [inline .github/fabricbot.json:3690]: I think `[Aa][Oo][Tt]` would work, but you could add word boundaries if you were feeling really fussy.  I don't think the parens or `+` are needed.

**Review by amcasey — APPROVED** [review]

File: .github/fabricbot.json (hunk 1 above)
Comment (eerhardt) [inline .github/fabricbot.json:3690]: What kind of regex engine is being used? .NET? If it is, could this be written:

```suggestion
                "bodyPattern": "(?i)(?:aot)",
```

Comment (amcasey) [reply .github/fabricbot.json:3690]: We clarified offline that they don't support options. 😢

Comment (eerhardt) [reply .github/fabricbot.json]: How else do you do case insensitive? is there a parameter for it? Seems like a major limitation.

Comment (mkArtakMSFT) [reply .github/fabricbot.json]: It is a major limitation indeed. Which they now know about !

**Pull request merged**